test/parsing/errors/recordInheritanceNotSupported-detailed.good

```diff
@@ -0,0 +1,4 @@
+─── error in recordInheritanceNotSupported.chpl:5 [RecordInheritanceNotSupported] ───
+  Inheritance is not currently supported for records.
+  thoughts on what record inheritance should entail can be added to https://github.com/chapel-lang/chapel/issues/6851
```

Comment:
Looks like this should be uppercased (`message` is the only command that doesn't auto-uppercase, and that's because it's _only_ shown in verbose mode).